modules/flowable-engine/src/test/java/org/flowable/engine/test/bpmn/event/compensate/CompensateEventTest.java

```diff
@@ -13,12 +13,6 @@
 
 package org.flowable.engine.test.bpmn.event.compensate;
 
-import static org.assertj.core.api.Assertions.assertThat;

```

Comment:
Please follow the project import ordering; see https://github.com/flowable/flowable-engine/blob/main/ide-settings/eclipse/Preferences-Java-CodeStyle-OrganizeImports.importorder

Reply:
I have already made the necessary changes